src/blocks/RulesBlock.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+
+function RulesModuleHeader() {
+  return (
+    <div className="rules-header">
+      <h3>📌 Pinned Rules</h3>
+      <a href="#" className="rules-link">
+        See all
+      </a>
+      <style jsx>{`
+        .rules-header {
+          display: flex;
+          justify-content: space-between;
+          align-items: center;
+          padding: 6px;
+          height: 1.2em;
+        }
+        .rules-link {
+          color: #888;
+          text-decoration: none;
+          height: 1.2em;
+          padding: 0px 6px;
+        }
+      `}</style>
+    </div>
+  );
+}
+
+function RuleDisplay({ rule }) {
+  return (
+    <li key={rule.index}>
+      <details className="details">
+        <summary className="summary">{rule.title}</summary>
```

Comment:
No need to add the `className` here. You can directly target these elements in the CSS with `summary {` or `details {` without the preceding dot.

If you see me not doing this in the codebase is because I'm also guilty of adding useless classes sometimes.